<ui version="4.0" >
 <author>Borys Jurgiel</author>
 <class>QgsPluginInstallerInstallingDialogBase</class>
 <widget class="QDialog" name="QgsPluginInstallerInstallingDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QGIS Python Plugin Installer</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >:/plugins/installer/qgis-icon.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>502</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Installing plugin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="labelState" >
     <property name="text" >
      <string>Connecting...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="alignment" >
      <set>Qt::AlignHCenter</set>
     </property>
     <property name="textDirection" >
      <enum>QProgressBar::TopToBottom</enum>
     </property>
     <property name="format" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>502</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Abort</set>
     </property>
     <property name="centerButtons" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
